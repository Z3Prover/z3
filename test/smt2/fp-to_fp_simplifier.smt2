
; Copyright (c) 2015 Microsoft Corporation
(set-logic QF_FP)

(simplify (( _ to_fp 11 53 ) roundNearestTiesToEven 1.0 0)) 

(simplify (( _ to_fp 8 24 ) roundNearestTiesToEven 1.0 0)) 

(simplify ((_ to_fp 11 53) roundNearestTiesToEven
			(( _ to_fp 8 24 ) roundNearestTiesToEven 1.0 0)))
