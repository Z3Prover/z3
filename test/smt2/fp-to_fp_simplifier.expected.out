(fp #b0 #b01111111111 #x0000000000000)
(fp #b0 #x7f #b00000000000000000000000)
(fp #b0 #b01111111111 #x0000000000000)
